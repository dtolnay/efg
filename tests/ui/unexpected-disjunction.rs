use efg::efg;

#[efg(unix + windows)]
struct S;

fn main() {}
