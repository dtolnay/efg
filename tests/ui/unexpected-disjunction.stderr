error: unexpected token
 --> tests/ui/unexpected-disjunction.rs:3:12
  |
3 | #[efg(unix + windows)]
  |            ^
